kcidb/__init__.py

```diff
@@ -1,29 +1,30 @@
-"""Kernel CI reporting"""
+"""Kernel CI reporting."""
 
 import sys
 import email
 import logging
 from kcidb.misc import LIGHT_ASSERTS
+
 # Silence flake8 "imported but unused" warning
-from kcidb import io, db, mq, orm, oo, monitor, tests, unittest, misc # noqa
+from kcidb import io, db, mq, orm, oo, monitor, tests, unittest, misc  # noqa
 
 
 # Module's logger
 LOGGER = logging.getLogger(__name__)
 
 
 class DatabaseNotInitialized(Exception):
-    """Database is not initialized exception"""
+    """Database is not initialized exception."""
+
     def __init__(self):
         super().__init__("Database is not initialized")
 
 
 class Client:
-    """Kernel CI reporting client"""
+    """Kernel CI reporting client."""
 
     def __init__(self, database=None, project_id=None, topic_name=None):
-        """
-        Initialize a reporting client
+        """Initialize a reporting client.
```

Comment:
Hmm, what was the `pydocstyle` error message that prompted this and other similar changes?

I don't particularly like it, as it makes the description harder to notice in a multi-line docstring. Perhaps there's another way?

Reply:
In particular I get these kinds of error 

```
/home/abhishek/Downloads/kernelci/kcidb/kcidb/__init__.py:17 in public method `__init__`:
        D107: Missing docstring in __init__
/home/abhishek/Downloads/kernelci/kcidb/kcidb/__init__.py:22 in public class `Client`:
        D400: First line should end with a period (not 't')

```

Reply:
Ah, I meant putting the first line on the same line with the opening `"""`.

It seems it's not necessary (from https://peps.python.org/pep-0257/#multi-line-docstrings):
>  The summary line may be on the same line as the opening quotes or on the next line.

So could you please keep the summary line on the next line, after the opening `"""`?

Reply:
Then it throws an error single line comments should end in one line ://

Reply:
Ah, forgot to mention that. Yes. A single-line docstring is fine to be on the same line with **both** the opening **and** closing `"""`.

However, when you have multi-line ones, you tend to miss the first line.

I.e. we want this:
```python
"""Single-line docstring."""
```
and this:
```python
"""
Multi-line docstring summary.

Multi-line docstring elaboration, if any.

Args:
    x: An argument
"""

Reply:
Okay